### PR TITLE
FI-569: Add chained search tests

### PR DESCRIPTION
**Pull request by Jammjammjamm**
This branch adds a test for the chained search parameters US Core requires for PractitionerRole (practitioner.name and practitioner.identifier). Although this test was added to the US Core generator, the test itself and its unit test are hard coded. Implementing chained search tests in a generic manner would require a much larger amount of effort and isn't necessary to support these two chained parameters.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-569
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/uscore_generator.rb [hunk 1]
```diff
@@ -306,6 +309,71 @@ def create_search_test(sequence, search_param)
         )
       end
 
+      def create_chained_search_test(sequence, search_param)
+        # NOTE: This test is currently hard-coded because chained searches are
```
Comment (arscan) [inline generator/uscore/uscore_generator.rb]: could you raise an error in the generator if sequence not for PractionerRole just in case more are added in a future version?

**Comment by arscan** [comment]
Could you rebase when you have a chance?  We can get this one in next.
